### PR TITLE
Document AnimationNodeTimeSeek with clarified usage and code example

### PR DESCRIPTION
**Pull request by smix8**
Document AnimationNodeTimeSeek with clarified usage and code example.

Issue was that users often ask "how can I play my animation from a certain position inside the AnimationTree?" but have no idea from the current documentation what the SeekNode can do and how it is used. The majority didn't even know what a seek command in this context is to begin with.

### CODE REVIEW

File: tutorials/animation/animation_tree.rst [hunk 1]
```diff
@@ -96,7 +96,30 @@ This node will execute a sub-animation and return once it finishes. Blend times
 Seek
 ^^^^
 
-This node can be used to cause a seek command to happen to any sub-children of the graph. After setting the time, this value returns to -1.
+This node can be used to cause a seek command to happen to any sub-children of the animation graph. Use this node type to play an ``Animation`` from the start or a certain playback position inside the ``AnimationNodeBlendTree``.
+
+After setting the time and changing the animation playback the seek node automatically goes into sleep mode on the next process frame by setting its ``seek_position`` value to ``-1.0``.
```
Comment (Calinou) [inline tutorials/animation/animation_tree.rst:101]:
```suggestion
After setting the time and changing the animation playback, the seek node automatically goes into sleep mode on the next process frame by setting its ``seek_position`` value to ``-1.0``.
```

File: tutorials/animation/animation_tree.rst [hunk 2]
```diff
@@ -96,7 +96,30 @@ This node will execute a sub-animation and return once it finishes. Blend times
...
+    # alternative
+    anim_tree["parameters/Seek/seek_position"] = 12.0
+
+ .. code-tab:: csharp
+    
+    // play child animation from start:
+    animTree.Set("parameters/Seek/seek_position", 0.0);
+    
+    // play child animation from 12 sec timestamp
+    animTree.Set("parameters/Seek/seek_position", 12.0);
```
Comment (Calinou) [inline tutorials/animation/animation_tree.rst:122]:
```suggestion
.. tabs::
 .. code-tab:: gdscript GDScript

    # Play child animation from the start.
    anim_tree.set("parameters/Seek/seek_position", 0.0)
    # Alternative syntax (same result as above).
    anim_tree["parameters/Seek/seek_position"] = 0.0

    # Play child animation from 12 second timestamp.
    anim_tree.set("parameters/Seek/seek_position", 12.0)
    # Alternative syntax (same result as above).
    anim_tree["parameters/Seek/seek_position"] = 12.0

 .. code-tab:: csharp

    // Play child animation from the start.
    animTree.Set("parameters/Seek/seek_position", 0.0);

    // Play child animation from 12 second timestamp.
    animTree.Set("parameters/Seek/seek_position", 12.0);
```

**Review by Calinou — DISMISSED** [review]

**Comment by smix8** [comment]
@Calinou
@akien-mga 
This was merged for class in Godot 3.3.
Now that Godot 3.3 is released I think it would be good if it was merged for the docs as well.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.3 and 3.2.

**Pull request merged**